lib/import-export-visitor.js

```diff
@@ -415,17 +414,11 @@ function getBlockBodyInfo(visitor, path) {
     // If parent is a Program, we may need to add a "use strict"
     // directive to enable const/let in Node 4.
     needToAddUseStrictDirective = visitor.enforceStrictMode;
-
     break;
 
   case "BlockStatement":
     body = parent.body;
-
-    if (hasOwn.call(parent, "start")) {
```

Comment:
It looks like it's to replace blocks like `WhileStatement`, `IfStatement`, and `ForInStatement`.
Is there a reason to not have a start?